easybuild/easyblocks/e/esmf.py

```diff
@@ -146,7 +145,7 @@ def install_step(self):
 
             cmd = "python setup.py build --ESMFMKFILE=%s/lib/esmf.mk " % self.installdir
             cmd += " && python setup.py install --prefix=%s" % self.installdir
-            run_cmd(cmd, log_all=True, simple=True, log_ok=True)
+            run_shell_cmd(cmd)
```

Comment:
```suggestion
            run_shell_cmd(cmd, fail_on_error=True)
```

Reply:
`fail_on_error=True` is the default: https://github.com/easybuilders/easybuild-framework/blob/5.0.x/easybuild/tools/run.py#L184

Reply:
d'oh